README.md

```diff
@@ -9,18 +9,289 @@
   </a>
 </p>
 
-## Features
+## Styles
 
-### Typescript
+<hr />
 
-#### Docs
+This project uses [TailwindCSS](https://tailwindcss.com/) and [styled-components](https://styled-components.com/docs) using the [twin.macro](https://github.com/ben-rogerson/twin.macro/#features) plugin.
 
-- Typescript: https://www.typescriptlang.org/docs/
+### Basic
 
-### Styled Components + Tailwind + Twin.macro
+Use the `tw` prop when there are no conditional styles, and you don't need custom CSS. **This should always be preferred!**
 
-#### Docs
+```tsx
+// Note: we still need to import the macro if we want to use the `tw` prop.
```

Comment:
What about:

```
/**
 * We still need to import the macro, because it provides the custom JSX transform
 * that enables the `tw` prop. If no import is present, the compiler will default to
 * react-jsx, which does not know how to handle the `tw` prop.
 * 
 * This behavior might change in the future.
 * @see https://github.com/ben-rogerson/twin.macro/issues/247
 */
import "twin.macro";
```

Reply:
👌